backend/src/modules/match-module/core/domain/event/MatchHasEnded.ts

```diff
@@ -0,0 +1,13 @@
+export class MatchHasEnded {
+  readonly occurredAt: Date;
+  readonly matchId: string;
+  readonly winner: string;
+  readonly looser: string | undefined;
```

Comment:
Jak to jest możliwe, że zakończony mecz bedzie miał zwycięzce a przegranego nie :) ? Tutaj nie powinno być undefined.

Reply:
pomyślałem o sytuacji w drzewku gdzie jest 6 graczy i pierwszy mecz ma tylko jednego zawodnika, ale teraz już widzę, że to się kłóci z moimi wcześniejszymi założeniami :P dzięki za zwrócenie uwagi!